app/services/host_inventory_api.rb

```diff
@@ -50,6 +50,12 @@ def system_profile(ids)
 
   private
 
+  def find_os_release(system_profile)
+    return [nil, nil] if system_profile['os_release'].blank?
+
+    system_profile['os_release'].split('.')
```

Comment:
How sure are we that `os_release` will exist?

Reply:
Line 54 should prevent that, or?

Reply:
Oh yeah, true. :)